src/cmake/build_OpenEXR.cmake

```diff
@@ -0,0 +1,33 @@
+# Copyright Contributors to the OpenImageIO project.
+# SPDX-License-Identifier: Apache-2.0
+# https://github.com/AcademySoftwareFoundation/OpenImageIO
+
+
+set_cache (OpenEXR_LOCAL_BUILD_VERSION 3.2.3 "OpenEXR version for local builds")
+set (OpenEXR_GIT_REPOSITORY "https://github.com/AcademySoftwareFoundation/OpenEXR")
+set (OpenEXR_GIT_TAG "v${OpenEXR_LOCAL_BUILD_VERSION}")
+
+build_dependency_with_cmake(OpenEXR
+    VERSION         ${OpenEXR_LOCAL_BUILD_VERSION}
+    GIT_REPOSITORY  ${OpenEXR_GIT_REPOSITORY}
+    GIT_TAG         ${OpenEXR_GIT_TAG}
+    CMAKE_ARGS
+        -D CMAKE_BUILD_TYPE=${CMAKE_BUILD_TYPE}
+        # Build static libs
+        -D BUILD_SHARED_LIBS=OFF
```

Comment:
On MSVC - building only static lib with `proj_add_compile_definitions (OPENEXR_DLL)` added in `src/cmake/externalpackages.cmake` results with unresolved symbols:
```error LNK2019: unresolved external symbol "__declspec(dllimport) ...```
Commenting out this line, resolves the problem but is seem the correct way to handle this, is to condition the check when `BUILD_LOCAL missing` option is active.

Reply:
Yeah, thanks, found that yesterday, it was really messing with me!

Reply:
Perfect! Glad you've tracked this one down.